source/components-and-apis.rst

```diff
@@ -293,6 +307,14 @@ TCP port 8000), see the following table.
    * - X.509 Identity Management
      - host:3000/api/v1
      - host:8000/x509/v1
+   * - Kafka Websocket
+     - host:8080/v1/ticket
+     - host:8000/kafka-ws/v1/ticket
+   * - Kafka Websocket
```

Comment:
I think you can change  "Kafka Websocket" to "Kafka WS (Kafka Websocket)" or just "Kafka WS"